packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_services/translations_service.dart

```diff
@@ -0,0 +1,22 @@
+import '../models/i18n/i18n_models.dart';
```

Comment:
Please move all translation related files in a single `lib` such as lib_translations